content/20.treatments.md

```diff
@@ -10,8 +10,60 @@ Additionally, there have been suggestions within the scientific community that n
+
+Tocilizumab 
+
+Background: A recent study carried out on a sample of 191 Covid-19 patients from Wuhan shows that the blood of patients who faced death contained about 74% more interleukin-6 (IL-6) compared to the survivors (Zhou et al. 2020).
+This indicates an inflammatory response out of control or what is called “cytokine storm”.
+In this context IL-6 plays a key role (Tisoncik et al. 2012).
+IL-6 is a pro-inflammatory cytokine belonging to the family of interleukins, immune system regulators, primarily responsible for immune cell differentiation and activation.
+In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system as well as the alveolar spaces, ensuring homeostasis.
+It has been already shown that high level of IL-6, during the infection of other Corona viruses (influenza, SARS COV), lead to the high concentration of other cytokines (TFN, IL-1𝛽) with the results of heavily damaging the endotelium of blood vessels and disrupting the balance between pro and anti-inflammatory response (Park et al. 2001).
+The holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death as per Acute Distress Respiratory Syndrome (ADRS) or Severe Acute Respiratory Syndrome (SARS) (Channappanavar and Perlman 2017), also caused by the new Corona virus . 
+
+Recently Chinese and Italian doctors have found that the the Tocilizumab (acterma, by Roche), a drug commonly used in the rheumatoid arthritis, is very effective in the cure of Covid-19 patients. 
+Specifically, Tocilizumab is a humanised monoclonal antibody that binds to the receptor of IL-6, de facto inhibiting the IL-6 immune cascade. 
+Chinese doctors have started a trial enrolling 188 patients o which 14 with severe lung disease have shown clear sings of improvements, according to their results. 
+Also, The AIFA (the Italian Drug Agency) approved the start of a new trial on March 19 recruiting patients at the initial stage of the infection. 
+Together with these independent trials, Roche, also in collaboration with the FDA, will start a randomised, double-blind, placebo-controlled phase III trial early April. 
+The trial will enroll 330 patients globally, which will be followed for 60 days upon use of the drug via injection to analyse its efficiency/safety (Biopharma-reporter.com).
+
+Antivirals
+
+1)	Interferone- β. 
+
+Background: Interferons (IFNs) are a family of cytokines crucial to activate the first (innate) immune system response against viral infections. 
+Interferons are classified into three categories based on their receptor specificity: type I, II and III (Tisoncik et al. 2012). 
+Specifically, IFNs I (IFN-𝛼 and 𝛽) and II (IFN-𝛾 ) induce the expression of antiviral proteins which bring the viral RNA to degradation (Spiegel et al. 2004). 
```

Comment:
I think type III interferons merit additional discussion here. Type III interferons are major contributors to antiviral responses at epithelial barriers like the lung.

This paper demonstrates that IFN-lambda restricts viral replication in the lung and GI tract of SARS-CoV infected animals. IFN-lambda receptor 1 knockout animals (Ifnlr1-/-; aka Il28ra-/-) had increased viral load in affected tissues compared with WT animals. Magnitude of effect was similar to interferon-alpha receptor (the type I IFN receptor) knockouts. Interestingly, it seems that both interferons contribute as double receptor knockouts had higher viral loads than either of the single knockouts... 
[](https://jvi.asm.org/content/84/11/5670#sec-11)

So why should we care about lambda interferons? First, Eiger biopharma just announced they will be testing pegylated-IFN lambda as a treatment for COVID-19. Second, there is quite a lot of literature to suggest that IFN lambda induces antiviral responses without inducing tissue inflammation/damage like the type I IFNs... I worked on lambda interferons for several years so I am happy to write more about this!

Will add more to the tocilizumab section as I find some free time. Is anyone writing about hydroxychloroquine? Coming from a rheum background I have lots of thoughts on HCQ and some of the original papers that came out...

Reply:
I can open an issue for hydroxychloroquine (and tag you), it's definitely important that we include it!

Reply:
I'm happy to write a quick paragraph on it.

Reply:
Thank you very much all for your suggestions. Will incorporate them all ASAP. I added a paragraph on Remdesivir in the meantime, I can collaborate on the paragraph on hydroxychloroquine as well.